h2o-core/src/main/java/water/util/Countdown.java

```diff
@@ -0,0 +1,60 @@
+package water.util;
+
+import java.io.Serializable;
+
+/**
+ * Simple countdown to encapsulate timeouts and durations.
+ * time_limit_millis <= 0 is interpreted as infinite countdown (no timeout)
+ */
+public class Countdown implements Serializable {
```

Comment:
should be Iced instead of Serializable (or nothing ;))

Reply:
well, I definitely forgot to provide empty constructor so Serializable wouldn't work... but is it really necessary to implement Iced for such simple classes (2 long fields)? I mean technically, the Weaver delegates to Java serialization for Serializable objects...
Just asking to personal info ;) switching to Iced anyway, thx for noticing.